plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1297,6 +1297,31 @@ public static function has_product_subscription( $product_id ) {
 		return ! empty( $subscription );
 	}
 
+	/**
+	 * Get the user's connected subscriptions that are installed on the current
+	 * site.
+	 *
+	 * @return array
+	 */
+	public static function get_installed_subscriptions() {
+		static $installed_subscriptions = null;
+
+		// Cache installed_subscriptions in the current request.
+		if ( is_null( $installed_subscriptions ) ) {
+			$auth    = WC_Helper_Options::get( 'auth' );
+			$site_id = isset( $auth['site_id'] ) ? absint( $auth['site_id'] ) : 0;
```

Comment:
Maybe we can return early here, after setting `$installed_subscriptions = []`, when `$auth['site_id']` isn't set.

Reply:
Good point. I'll update it.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/49717/commits/132c8dff227d21975815bb5c42795b543df17181.